layers/meta-tegrademo/conf/distro/tegrademo.conf

```diff
@@ -75,4 +75,4 @@ INHERIT += "reproducible_build"
 BB_SIGNATURE_HANDLER ?= "OEEquivHash"
 BB_HASHSERVE ??= "auto"
 
-LICENSE_FLAGS_WHITELIST += "commercial_faad2"
+LICENSE_FLAGS_WHITELIST += "commercial_faad2 commercial"
```

Comment:
This should be a separate commit. Also, I'd prefer minimizing the whitelist, restricting it to just the required packages (in this case, `ffmpeg`, and `x264` due to the dependency in the `ffmpeg` build). That means another commit with a bbappend for `opencv` that adds `libav` to its PACKAGECONFIG would also be needed, since the opencv recipe only looks for the full `commercial` whitelist flag.